zetaclient/bitcoin/bitcoin_client.go

```diff
@@ -1296,41 +1310,50 @@ func (ob *BTCChainClient) checkTSSVin(vins []btcjson.Vin, nonce uint64) error {
 	return nil
 }
 
+// DecodeP2WPKHVout decodes receiver and amount from P2WPKH output
+func (ob *BTCChainClient) DecodeP2WPKHVout(vout btcjson.Vout) (string, int64, error) {
```

Comment:
Can we add tests for this method?

Reply:
This func extracts existing logics from existing `checkTSSVout`. But I can give it a minimum unit test.

Reply:
https://github.com/zeta-chain/node/pull/1790/commits/cf72c1bfc036951b792b8d3d7b327fbb7b53cc32